assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -25,18 +25,37 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import { useAPI } from '../../api';
 import { OutputPage } from '../../../output';
+import { useConfig } from '../../config';
 
 /**
  * Creates AMP HTML markup for saving to DB for rendering in the FE.
  *
- * @param {Object} pages Object of pages.
+ * @param {Object} story Story object.
+ * @param {string} story.featuredMediaUrl Featured media URL.
+ * @param {Array<Object>} pages List of pages.
+ * @param {Object} metadata Metadata.
+ * @param {string} metadata.publisher Publisher name.
+ * @param {string} metadata.publisherLogo Publisher logo.
  * @return {Element} Markup of pages.
  */
-const getStoryMarkupFromPages = (pages) => {
-  const markup = pages.map((page) => {
-    return renderToString(<OutputPage page={page} />);
-  });
-  return markup.join('');
+const getStoryMarkup = (story, pages, metadata) => {
+  // TODO: get different image sizes for featured media.
+
+  return renderToString(
+    <amp-story
+      standalone="standalone"
+      publisher={metadata.publisher}
+      publisher-logo-src={metadata.publisherLogo}
+      title={story.title}
+      poster-portrait-src={story.featuredMediaUrl}
```

Comment:
Shouldn't we have different image sizes here? Surely portrait / square / landscape would not work here.

Reply:
It's an open todo listed in the code and in the PR description above.